src/ast/npc.cpp

```diff
@@ -319,16 +319,10 @@ namespace npc
         {
             return Err(bits[2].span.error_str("in |component 3| expect 1 ,component,s"_s));
         }
-        assert(bits[3].data[0].data == "32767"_s);
```

Comment:
the `|32767` was constant (ALL global npcs needed this) so it made sense to remove it